packages/core-utils/src/optimism/constants.ts

```diff
@@ -30,6 +30,10 @@ export const predeploys = {
   ETH: '0x4200000000000000000000000000000000000486',
 }
 
+export const L1Predeploys = {
+  L1TonAddress: '0x2be5e8c109e2197d077d13a82daead6a9b3433c5',
```

Comment:
What do you think if L1TonAddress is configured when init cross-domain-messenger? I think this address is not contant

Reply:
yes, let me update this